src/arch/riscv/RiscvISA.py

```diff
@@ -58,3 +84,13 @@ class RiscvISA(BaseISA):
     riscv_type = Param.RiscvType("RV64", "RV32 or RV64")
 
     enable_rvv = Param.Bool(True, "Enable vector extension")
+    vlen = Param.RiscvVectorLength(
+        256,
```

Comment:
According to RVV spec, v extension implies VLEN=128 by default

Reply:
Yes, we should put 128 but that would break the CI tests that depend on checkpoints taken for VLEN=256. I think we should try to get this in asap and later we can fix it.

Reply:
Can you point me to where in the spec it says 128? Also, what is the importance of this default value? You can always override this in your standard library board or from your config script.

Personally, I strongly prefer to merge this now and we can discuss changing the default later.

Reply:
@powerjg please read the spec
https://github.com/riscv/riscv-v-spec/blob/v1.0/v-spec.adoc#183-v-vector-extension-for-application-processors

```
The V vector extension requires Zvl128b.
```

Reply:
If I'm not mistaken, `Zvl128b` means VLEN >= 128b. I think the default VLEN = 256b is sensible as long as we're clear about the VLEN we're using.

Reply:
Nope `Zvl128b` means VLEN=128b, and the spec accepts maximum of VLEN=65536b (`zvl65536b`)

Reply:
I am *so* confused... 

First, thanks @jerryzj for the review and pointing to the spec.

Are you saying that we cannot create a machine that has a VLEN other than 128b? That doesn't make sense to me. In theory, I can build/buy a RISC-V processor with a VLEN of anything between 128 and 65536. Why does it matter what gem5 sets its default to? Anyone is able to change this value to anything they want *without rebuilding gem5.*

As far as I can tell, the spec doesn't say anything about "default" values.

Could you explain why changing the default from 256 to 128 will benefit the users of gem5?

+1 for making it clear what VLEN is, but I don't know if adding more noise to stdout is the right way to do it.